### DISCUSSION TITLE
IPERF3 test udp i do not see packet coming on server side why ?  i lunch this command on my client side "Iperf3.exe -u -c “server IP” -b 2G -t 120"

### DISCUSSION THREAD

**Discussion by fabriziosala130** [esnet/iperf#1465] [Q&A]
_NOTE: The iperf3 issue tracker is for registering bugs, enhancement
requests, or submissions of code.  It is not a means for asking
questions about building or using iperf3.  Those are best directed
towards the Discussions section for this project at
https://github.com/esnet/iperf/discussions
or to the iperf3 mailing list at iperf-dev@googlegroups.com.
A list of frequently-asked questions
regarding iperf3 can be found at http://software.es.net/iperf/faq.html._

# Context

* Version of iperf3:

* Hardware:

* Operating system (and distribution, if any):

_Please note: iperf3 is supported on Linux, FreeBSD, and macOS.
Support may be provided on a best-effort basis to other UNIX-like
platforms.  We cannot provide support for building and/or running
iperf3 on Windows, iOS, or Android._

* Other relevant information (for example, non-default compilers,
  libraries, cross-compiling, etc.):

_Please fill out one of the "Bug Report" or "Enhancement Request"
sections, as appropriate. Note that submissions of bug fixes, new
features, etc. should be done as a pull request at
https://github.com/esnet/iperf/pulls_

# Bug Report

* Expected Behavior

* Actual Behavior

* Steps to Reproduce

* Possible Solution

# Enhancement Request

* Current behavior

* Desired behavior

* Implementation notes

**Comment by fabriziosala130** [comment]
IPERF3 test udp i do not see packet coming on server side why ? i lunch this command on my client side "Iperf3.exe -u -c “server IP” -b 2G -t 120"

**Reply by bmah888** [reply]
For anyone to answer your question they'll need some more information.

Was the server actually running, if so, what were its command-line arguments? 

This looks like it's Windows, is that correct?

What version of iperf3? 

What output did you get on the client side?

Does a TCP test work?

Are there any host-based firewalls on either the client or server that might be blocking the UDP test traffic?
